script/lib/native_tests.py

```diff
@@ -0,0 +1,274 @@
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+
+SOURCE_ROOT = os.path.abspath(
+    os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+VENDOR_DIR = os.path.join(SOURCE_ROOT, 'vendor')
+PYYAML_LIB_DIR = os.path.join(VENDOR_DIR, 'pyyaml', 'lib')
+sys.path.append(PYYAML_LIB_DIR)
+import yaml  #pylint: disable=wrong-import-position,wrong-import-order
+
+
+class Verbosity:
+  CHATTY = 'chatty'  # stdout and stderr
+  ERRORS = 'errors'  # stderr only
+  SILENT = 'silent'  # no output
+
+  @staticmethod
+  def get_all():
+    return Verbosity.__get_all_in_order()
+
+  @staticmethod
+  def __get_all_in_order():
+    return [Verbosity.SILENT, Verbosity.ERRORS, Verbosity.CHATTY]
+
+  @staticmethod
+  def __get_indices(*values):
+    ordered = Verbosity.__get_all_in_order()
+    indices = map(ordered.index, values)
+    return indices
+
+  @staticmethod
+  def ge(a, b):
+    """Greater or equal"""
+    a_index, b_index = Verbosity.__get_indices(a, b)
+    return a_index >= b_index
+
+  @staticmethod
+  def le(a, b):
+    """Less or equal"""
+    a_index, b_index = Verbosity.__get_indices(a, b)
+    return a_index <= b_index
```

Comment:
The impl in the previous four methods seems like overkill compared to making something like

```py
class Verbosity:
  SILENT = 0
  ERRORS = 1
  CHATTY = 2

...
@def ge(a, b):
  """Greater or equal"""
  return a >= b
```

Reply:
I need `Verbosity` level to be human readable strings to use them as script arguments. `make-script.py --verbosity 2` doesn't make any sense.

Reply:
Another option is to actually instantiate objects of `Verbosity` class, initialising them with simple string values, like `verbosity = Verbosity(Verbosity.SILENT)` but it looks like overengineering to me =)